app/templates/courses/enrollments-view.jade

```diff
@@ -31,79 +31,105 @@ block content
           a.btn.btn-primary.btn-lg(href="/teachers/update-account") Upgrade to teacher account
       
     .container.m-t-5
+      .pull-right
+        span.glyphicon.glyphicon-question-sign
+        =' '
+        a#how-to-enroll-link(data-i18n="teacher.how_to_enroll")
       h3(data-i18n='teacher.enrollments')
-      h4
+      h4#enrollments-blurb
         span(data-i18n='teacher.enrollments_blurb_1')
-        span  2&ndash;8 
+        span  2&ndash;#{view.state.get('totalCourses')} 
         span(data-i18n='teacher.enrollments_blurb_2')
         
+      - var available = view.state.get('prepaidGroups').available
+      - var pending = view.state.get('prepaidGroups').pending
+      - var anyPrepaids = available || pending
```

Comment:
There's a subtle bug that doesn't affect much in the end here, but is why I especially don't like using implied booleans. The initial state has available and pending set to `[]`, which are truthy, not falsey. That means the wrong thing renders until they get overwritten with `undefined`s which are falsey.

Using `available.length > 0` might take a little more typing but is immune to this kind of bug.